src/pages/Statistics/Stat.tsx

```diff
@@ -0,0 +1,44 @@
+import Card from "../../components/Card";
+import { mean, median, mode, std, min, max } from "mathjs";
+
+/**
+ * Custom component for displaying stat cards
+ * @param values list of happiness values used in calculating statistic
+ * @param statName name of statistic value
+ * @param key key for statistic
+ * @returns
+ */
+
+export default function Stat({
+  values,
+  statName,
+}: {
+  values: number[];
+  statName: number;
+}) {
+  let statLabel = ["Average", "Median", "Mode", "STD", "Minimum", "Maximum"];
+  let calculatedStats = [
+    mean(values).toFixed(2),
+    median(values).toFixed(2),
+    // @ts-ignore
+    // for some reason, ts isn't working with this
```

Comment:
maybe it's because values could be empty here? try to get ts to work if you can.

Reply:
I fixed the issue with the standard deviation ts stuff, but for some reason, the mode is represented by a number (but sometimes it's a list of numbers). If I assume it's always a number, it can break if there's multiple modes. I'm not completely sure how to fix it; it could just be a library issue.

Reply:
I tried casting to a number list, but that didn't work out

Reply:
interesting, the library supports TypeScript so I feel like you should be able to type it. Not too big of a deal for now though.